# pas de mise à jour

challenge reverse pyc en 3.10

à ma connaissance (et pour l'instant) le seul decompilateur qui supporte le 3.10 ne reconnait pas les pattern match... comme c'est dommage...

le fichier `sol.py` est une solution et le fichier `chall.py` est le fichier source du challenge.

Bien sûr les participants n'avaient accès qu'à `chall.pyc`.
